skeleton/Gemfile

```diff
@@ -27,5 +25,5 @@ end
 group :system_tests do
   gem 'beaker'
   gem 'beaker-rspec'
-  gem 'simp-beaker-helpers', ENV.fetch('SIMP_BEAKER_HELPERS_VERSION', ['>= 1.10.7', '< 2'])
+  gem 'simp-beaker-helpers', ENV.fetch('SIMP_BEAKER_HELPERS_VERSION', ['>= 1.12.1', '< 2.0'])
```

Comment:
Did something require the `2.0` instead of just `2`?  It's using a normal comparison operator, and not `~>`